src/py/pyodide/webloop.py

```diff
@@ -627,6 +628,20 @@ def set_event_loop(self, loop: Any) -> None:
         self._default_loop = loop
 
 
+_orig_run = asyncio.run
+
+
+@wraps(_orig_run)
+def _run(main, *, debug=None, loop_factory=None):
+    from pyodide_js._api import config
+
+    if loop_factory is None and config.enableRunUntilComplete:
```

Comment:
Just curious if there is something we can do when `loop_factory` is passed.

Reply:
I think our options are we could ignore it or we could do this.